execute_two.py

```diff
@@ -24,18 +24,18 @@ def __init__(self, menu, plots=None):
 
 DirIn,field=parser.parse(sys.argv)
 
-Vradar = Variable(None)
+
 
 
 app = QtGui.QApplication(sys.argv)
 
-MainMenu = Menu(Vradar, DirIn, name="Menu") #initiate Vradar
+MainMenu = Menu(DirIn, name="Menu") #initiate Vradar
+Vradar = MainMenu.Vradar
 
 Vtilt = Variable(0)
 Vtilt2 = Variable(0)
-Vlims = Variable(None)
-plot1 = Display(Vradar, Variable(field), Vtilt, Vlims, name="Display1", parent=MainMenu)
-plot2 = Display(Vradar, Variable(field), Vtilt2, Vlims, name="Display2", parent=MainMenu)
+plot1 = Display(Vradar, Variable(field), Vtilt, name="Display1", parent=MainMenu)
+plot2 = Display(Vradar, Variable(field), Vtilt2, name="Display2", parent=MainMenu)
 
 app.exec_()
 
```

Comment:
I think it's a good idea to clean this script up.  
A thought: Something we might consider is building and function that would return Variable instances such as Vtilt (and potential other in the future) and use these for Display calls.  We could have a default method and then it might be scalable for the future.  I'm thinking of adding to the main menu a **Create Display** menu that pops up an additional window.  Then this would remove the need for multiple execute scripts or we could then call this the execute script.  Thoughts?